deploy/clowdapp.yaml

```diff
@@ -20,6 +20,9 @@ objects:
       # Details about running pod
       podSpec:
         image: ${IMAGE}:${IMAGE_TAG}     
+        command:
```

Comment:
Do we need to pass the environment vars here too like in the Makefile?

Reply:
The environment variables are below this.  Take a look at line 46.